workflows/databases/expressionatlas.nf

```diff
@@ -35,16 +39,18 @@ process fetch_lookup {
 
 
 process parse_tsvs {
-  memory 50.GB
+  memory 92.GB
```

Comment:
This is kinda huge. Let's chat about how to use a smaller amount of RAM.

Reply:
I can split the process in two and maybe also split the set of all genes in expressionatlas up to parallelise this step while also reducing memory. It's only using that much memory because it does an absolutely huge join on two dataframes in the lookup step, so toning that down a bit should help a lot